packages/devtools_app/lib/src/debugger/program_explorer.dart

```diff
@@ -377,32 +378,53 @@ class ProgramExplorer extends StatelessWidget {
         if (!initialized) {
           body = const CenteredCircularProgressIndicator();
         } else {
+          const fileExplorerHeader = AreaPaneHeader(
```

Comment:
either remove this const since it is only used once, or keep it and change `needsTopBorder` to `serviceManager.connectedApp.isDartWebAppNow` so it can be used as part of the `Column` and the `FlexSplitColumn`